static/styles/settings.css

```diff
@@ -483,6 +483,16 @@ input[type=checkbox].inline-block {
     height: 75px;
 }
 
+#settings_page .custom_user_field input[type=number]::-webkit-outer-spin-button,
+#settings_page .custom_user_field input[type=number]::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+}
+
+#settings_page .custom_user_field input[type=number] {
+    -moz-appearance:textfield;
+}
```

Comment:
If we want we can apply this css to all numeric input fields, not just custom profile data.